### PR TITLE
Add and modify module files to compile on AWS

### PR DESCRIPTION
**Pull request by weihuang-jedi**
Add and modify module files to compile GSI-utils on AWS,
resolve issure: #135 

https://github.com/NOAA-EMC/GSI-Monitor/issues/135

### CODE REVIEW

File: modulefiles/noaacloud-run.intel.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
+help([[
+]])
+
+prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
+
+local stack_intel_ver=os.getenv("stack_intel_ver") or "2021.3.0"
+local stack_impi_ver=os.getenv("stack_impi_ver") or "2021.3.0"
+local grads_ver=os.getenv("grads_ver") or "2.2.3"
+local perl_ver=os.getenv("perl_ver") or "5.38.0"
```
Comment (EdwardSafford-NOAA) [inline modulefiles/noaacloud-run.intel.lua]: Is perl_ver needed if there's no perl module loaded here?

**Comment by weihuang-jedi** [comment]
Ed,

Nice catch!
perl is not needed, and therefor now need to define here.
I have the line removed and pushed in a new change.

Thank you,

Wei

**Review by EdwardSafford-NOAA — APPROVED** [review]
Looks good, thanks.

**Comment by weihuang-jedi** [comment]
Thank you!

**Pull request merged**